docs/index.rst

```diff
@@ -21,7 +21,7 @@ Welcome to MetalK8s's documentation!
 
    installation-guide/index
    developer/index
-
+   changenodehostname
```

Comment:
forgot to change that in my commit. Now named: change_node_hostname.
Even should be renamed: changing_node_hostname

Reply:
@lucieleonard :

Talking about commits: we try to stick to issue # in the body, and the title as the area of intervention + a short description.

We'd rewrite your commit with something along the lines of 

```
docs: rework hostname change procedure

* procedure becomes a list of steps
* [...]

Issue: #1162 
```

Additionally, it would be great to have a separate commit for the rename, so that the diff on the commit that actually makes changes to the file is not just 1 file deleted and 1 added, but rather the actual rework that happened.